File: 5-matematica/1170/1170.c
#include <stdio.h>

int main(void) {
    int blobs, dias;
    float comida;
    scanf("%d", &blobs);
    for (blobs; blobs>0; blobs--) {
        scanf("%f", &comida);
        dias = 0;
        while (comida > 1) {
            comida /= 2;
            dias++;
        }
        printf("%d dias\n", dias);
    }
    return;
}

File: 5-matematica/1170/1170.lua
local caso, comida, dias
for caso = tonumber(io.read()), 1, -1 do
    comida = tonumber(io.read())
    dias = 0
    while comida>1 do
      comida = comida/2
      dias = dias + 1
    end
    print(string.format("%d dias", dias))
end